client/src/Layouts/RightSideBar.js

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from "react";
 
 import PlusImg from "../Assets/img/plus-math-30.png";
-import PlayImg from "../Assets/img/play.png";
 import Popup from "../Components/Popup";
+import Pomodoro from "../Components/Pomodoro";
 
 const RightSideBar = ({ user, SERVER_URL }) => {
```

Comment:
Might be worth looking at putting the SERVER_URL into context or some other storage?

Reply:
I was going to create a new branch for deployment updates and action this on that branch after we talked about it, I have to admit I'm pretty curious to have a play with the useContext hook :)